doc/reference/reference_rock/vshard/vshard_ref.rst

```diff
@@ -214,7 +214,7 @@ Router public API
 * :ref:`vshard.router.callbre(bucket_id, function_name, {argument_list}, {options}) <router_api-callbre>`
 * :ref:`vshard.router.route(bucket_id) <router_api-route>`
 * :ref:`vshard.router.routeall() <router_api-routeall>`
-* :ref:`vshard.router.bucket_id(key) <router_api-bucket_id>`
+* :ref:`vshard.router.bucket_id_(key) <router_api-bucket_id_strcrc32>`
```

Comment:
Old function didn't change its name. It is still `bucket_id`, not `bucket_id_`.

Reply:
thanks! It is a typo